kuma/javascript/src/payments/components/subscription-form.jsx

```diff
@@ -9,45 +9,11 @@ import GAProvider, {
 } from '../../ga-provider.jsx';
 import UserProvider from '../../user-provider.jsx';
 import { getCookie } from '../../utils';
+import { ErrorComponent } from './errors.jsx';
+import useScriptLoading from './useScriptLoading.js';
```

Comment:
In order to mock this function, I had to move it to its own file. I tried other ways around this but couldn't get it to work.